packages/spindle-icons/scripts/icon-doc.ts

```diff
@@ -1,11 +1,14 @@
 import { promises as fs } from 'fs';
+import { relative } from 'path';
 
+const DOC_DIR = 'docs';
 const SVG_DIR = 'dist/svg';
 
 const markdown = (files: string[]): string => {
+  const svgPath = relative(DOC_DIR, SVG_DIR);
   const icons = files.map((file) => {
     const name = file.replace('.svg', '');
-    return [`| ${name} | ![](${SVG_DIR}/${file}) |`].join('');
+    return [`| ${name} | ![](${svgPath}/${file}) |`].join('');
```

Comment:
Array#joinをしている意味があまりない気がするのですが、以下だとダメですかね？（おそらく元々はjoinで綺麗になるようなコードだったんだと予想してます）

```suggestion
    return `| ${name} | ![](${svgPath}/${file}) |`;
```

Reply:
うわ・・これは初期の"何か"の名残ですね・・ｗ

何かはもちろん覚えてないですが:grin: